### DISCUSSION TITLE
calculation for filer size?

### DISCUSSION THREAD

**Discussion by peedy2495** [seaweedfs/seaweedfs#2664] [General]
I designated four 2T SSDs, each with a volume-service.
Max is set to 58 per Volume Service.
Replica was set to 001 (all are in the same rack).

Why is a filer mount telling about 350GB are usable?

**Comment by chrislusf** [comment]
The replication can be different for different requests. The max volumes setting is for maximum physical volume count, but only a few volumes are created upfront.

Currently, the FUSE mount max size is calculated as
```
 volume_size * created_logic_volume_count
```

You are looking for could be:
```
 volume_size * max_physical_volume_count / default_replication
```

**Comment by peedy2495** [comment]
So the available size will increase in volume-size steps, when the used volumes reached their maximum capacity until the max volume count of each volume server has been reached?
Certainly regarding any additional replication methods ;-)

**Comment by peedy2495** [comment]
Ok, i tested the behaviour with big files ... et voila ... doubled the available size :-)
This should be a good hint in the documentation ;-)
Is it possible to preallocate the maximum number of volumes?
...  to get the real available size to have a situation awareness for applications, when they're importing/producing mass data?
